app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -18,21 +18,23 @@ export const NOTES_FORM_VALIDATION_SCHEMA = yup.object().shape({
     .required(t("required.entity", { entity: "Description" })),
   assignedContact: yup
     .object()
-    .test(
-      "isValidLabel",
-      t("required.entity", { entity: "Assigned Contact" }),
-      value =>
-        value &&
-        value.label &&
-        typeof value.label === "string" &&
-        value.label.trim() !== ""
-    ),
+    .nullable()
+    .shape({
+      label: yup
+        .string()
+        .trim()
+        .required(t("required.entity", { entity: "Assigned Contact" })),
+      value: yup
+        .string()
+        .trim()
+        .required(t("required.entity", { entity: "Assigned Contact" })),
+    }),
   tags: yup.array().min(1, t("required.entities", { entities: "Tags" })),
```

Comment:
Shouldn't these be required as well?

Reply:
Yes, I have updated that here #18 
I misunderstood when you said to make it `nullable` later I understood what you meant.